testing/jormungandr-integration-tests/src/jormungandr/grpc/client_tests.rs

```diff
@@ -344,3 +344,109 @@ pub async fn get_fragments() {
     let client = Config::attach_to_local_node(config.get_p2p_listen_port()).client();
     println!("{:?}", client.get_fragments(vec![fragment_id.into()]).await);
 }
+
+// L1021 PullBlocks correct hashes
```

Comment:
I've been wondering what do these L labels mean? Check with @dkijania that we are not cargo-culting them in some wrong way.

Reply:
only informational, so it can stay like this. this was inhouse mapping between manual test and automated ones.